src/utils/celery_tasks.py

```
@@ -0,0 +1,174 @@
+            results.append(response)
+
+        if webhook_config:
+            webhook = CustomWebhook(WebhookConfig(**webhook_config))
+            logger.info(
+                f"Posting results to the webhook configured at {webhook.config.endpoint}"
+            )
+            res = webhook.post_result({"results": results, "session_id": session_id})
+            logger.info(f"Results posted to the webhook with res: {str(res)}")
+
+        return {"result": results, "session_id": session_id}
+    except Exception as err:
```

Ruff (0.11.9)

Local variable `err` is assigned to but never used

Remove assignment to unused variable `err`

(F841)